addon_service/external_storage_service/models.py

```diff
@@ -1,106 +1,23 @@
-from django.contrib.postgres.fields import ArrayField
-from django.core.exceptions import ValidationError
 from django.db import models
 
-from addon_service.addon_imp.models import AddonImpModel
-from addon_service.common import known_imps
-from addon_service.common.base_model import AddonsServiceBaseModel
-from addon_service.common.credentials_formats import CredentialsFormats
-from addon_service.common.service_types import ServiceTypes
-from addon_service.common.validators import (
-    validate_credentials_format,
-    validate_service_type,
-    validate_storage_imp_number,
-)
+from addon_service.abstract.external_storage.models import ExternalService
+from addon_service.common.validators import validate_storage_imp_number
 
 
-class ExternalStorageService(AddonsServiceBaseModel):
-    display_name = models.CharField(null=False)
+class ExternalStorageService(ExternalService):
     int_addon_imp = models.IntegerField(
         null=False,
         validators=[validate_storage_imp_number],
         verbose_name="Addon implementation",
     )
```

Comment:
Can this be moved to the `ExternalService` class as well? I assume each external service, regardless of type, should have a addon implementation and therefore a int representing that implementation.

Reply:
This field has a validator which is checking whether this is **Storage** implementation, so it cannot be common for all external services.